examples/distributed-pytorch/train.yaml

```diff
@@ -11,10 +11,14 @@ setup: |
     cd examples
     git filter-branch --prune-empty --subdirectory-filter distributed/minGPT-ddp
     # SkyPilot's default image on AWS/GCP has CUDA 11.6 (Azure 11.5).
+    uv venv --python 3.10
+    source .venv/bin/activate
     uv pip install -r requirements.txt "numpy<2" "torch==1.12.1+cu113" --extra-index-url https://download.pytorch.org/whl/cu113
 
 run: |
-    cd examples/mingpt
+    cd examples
+    source .venv/bin/activate
+    cd mingpt
```

Comment:
Do we also need to update https://github.com/skypilot-org/skypilot/blob/a60fe415eeb70022421d85a6bb88c79111710385/docs/source/running-jobs/distributed-jobs.rst?plain=1#L21-L41?